handle.go

```diff
@@ -28,6 +28,8 @@ type TaskHandle struct {
 	userCPUStats   *stats.CpuStats
 	systemCPUStats *stats.CpuStats
 
+	collectionInterval time.Duration
```

Comment:
Is there a specific why the driver would need to store the collection interval? The value would be passed from Nomad based on the client config, so it seems very local to `runStatsEmitter`.

Reply:
Ahh that makes sense, thanks! I missed noticing that it was crossing function boundaries 😅